yt_dlp/extractor/qqmusic.py

```diff
@@ -304,33 +369,21 @@ def _real_extract(self, url):
             note='Download toplist page',
             query={'type': 'toplist', 'topid': list_id, 'format': 'json'})
 
-        entries = [self.url_result(
-            'https://y.qq.com/n/yqq/song/' + song['data']['songmid'] + '.html', 'QQMusic',
-            song['data']['songmid'])
-            for song in toplist_json['songlist']]
+        entries = self._extract_entries(toplist_json, ('songlist', ..., 'data'))
+
+        list_name = join_nonempty(*traverse_obj(toplist_json, ((('topinfo', 'ListName'), 'update_time'),)), delim=' ')
+        list_description = traverse_obj(toplist_json, ('topinfo', 'info'))
```

Comment:
can we inline these?

the trailing comma in the `list_name` traversal can be removed

Reply:
https://github.com/yt-dlp/yt-dlp/pull/9768/commits/a4c83434337aedcbae2783a6c0ead0440b3a08a2
It needs trailing comma to let Python recognize outer `()` as one-elem tuple so it traverse to list of `toplist_json['topinfo']['ListName']` and `toplist_json['update_time']`.

Reply:
ah, mb I misread the traversal. IMO it's more readable to add a `None` to the path instead of only trailing comma